lens/util/repo.go

```diff
@@ -41,44 +41,44 @@ var log = logging.Logger("lily/lens")
 // GetMessagesForTipset returns a list of messages sent as part of pts (parent) with receipts found in ts (child).
 // No attempt at deduplication of messages is made. A list of blocks with their corresponding messages is also returned - it contains all messages
 // in the block regardless if they were applied during the state change.
-func GetExecutedAndBlockMessagesForTipset(ctx context.Context, cs *store.ChainStore, sm *stmgr.StateManager, next, current *types.TipSet) (*lens.TipSetMessages, error) {
+func GetExecutedAndBlockMessagesForTipset(ctx context.Context, cs *store.ChainStore, sm *stmgr.StateManager, current, executed *types.TipSet) (*lens.TipSetMessages, error) {
 	ctx, span := otel.Tracer("").Start(ctx, "GetExecutedAndBlockMessagesForTipSet")
 	defer span.End()
-	if !types.CidArrsEqual(next.Parents().Cids(), current.Cids()) {
-		return nil, xerrors.Errorf("child tipset (%s) is not on the same chain as parent (%s)", next.Key(), current.Key())
+	if !types.CidArrsEqual(current.Parents().Cids(), executed.Cids()) {
```

Comment:
This is enforcing a previous/next relationship between the tipsets. I don't really find the name `executed` to be useful. Why can't it be `parent` since it must be the tipset whose blocks are in the parent collection of the current tipset. It helps with the mental model that these tipsets are closely related: one follows another.

Reply:
If not, the text in the next error needs adjusting

Reply:
I felt `executed` was more descriptive as (I think) it aligns well with expected consensus: 
- it reminds me all messages it contains have been applied(executed) on the chain (to produce `current`)
- reminds me that `current` is an un-executed tipset. (almost call `current` `unexecuted` instead..)

Child and Parent just makes me think of their relation, one follows another, and doesn't describe the nature of EC as well.